repos.yaml

```diff
@@ -266,6 +266,11 @@ repositories:
     maintainers:
       - email: kerkhove.tom@gmail.com
         name: Tom Kerkhove
+  - name: kir4h
+    url: https://kir4h.github.io/charts
+    maintainers:
+      - name: Andres Alvarez 
+        email: 1671935+kir4h@users.noreply.github.com
```

Comment:
We need a real routable to contact you in case we need to contact you about something related to the repository.

Reply:
Added routable e-mail